packages/react/src/components/DataTable/TableHeader.js

```diff
@@ -141,6 +144,11 @@ TableHeader.propTypes = {
    */
   colSpan: PropTypes.number,
 
+  /**
+   * Supply an id to the th element.
+   */
+  headerId: PropTypes.string,
```

Comment:
```suggestion
  id: PropTypes.string,
```

Would it make sense to use `id` as the name of the prop or is there something that would not work with that name? 👀

Reply:
Your wish has been granted!